### PR TITLE
[FL-2125] Cli: input_dump

### PR DESCRIPTION
**Pull request by DrZlo13**
# What's new

- Return of the famous input_dump, now compatible with Cli and the fastest*. (*among others input_dump)

# Verification 

- Run input_dump in Cli, press any buttons on flipper.

# Checklist (do not modify)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

**Review by gornekich — DISMISSED** [review]

**Review by lomalkin — DISMISSED** [review]

File: applications/applications.c [hunk 1]
```diff
@@ -198,6 +199,8 @@ const FlipperOnStartHook FLIPPER_ON_SYSTEM_START[] = {
 #ifdef APP_UNIT_TESTS
     unit_tests_cli_init,
 #endif
+
+    input_dump_cli_init,
```
Comment (skotopes) [inline applications/applications.c]: tie with input srv

File: applications/debug_tools/input_dump_cli.c [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#include <cli/cli.h>
+#include <input/input.h>
+
```
Comment (skotopes) [inline applications/debug_tools/input_dump_cli.c]: maybe move to input srv?

**Review by skotopes — CHANGES_REQUESTED** [review]

File: applications/debug_tools/input_dump_cli.c (hunk 2 above)
Comment (DrZlo13) [reply applications/debug_tools/input_dump_cli.c]: Done in ecf0dc4

File: applications/applications.c (hunk 1 above)
Comment (DrZlo13) [reply applications/applications.c]: Done in ecf0dc4

File: applications/input/input_cli.c [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    furi_pubsub_unsubscribe(input_events, input_subscription);
+    furi_record_close("input_events");
+    osMessageQueueDelete(input_queue);
+}
+
+void input_cli_init() {
+    Cli* cli = furi_record_open("cli");
+    cli_add_command(cli, "input_dump", CliCommandFlagParallelSafe, input_dump, NULL);
```
Comment (skotopes) [inline applications/input/input_cli.c]: and now just move it to srv start code ;)

**Review by skotopes — CHANGES_REQUESTED** [review]

File: applications/input/input_cli.c (hunk 3 above)
Comment (skotopes) [reply applications/input/input_cli.c]: Sorry ) I were needing 2nd steps to realize that

Comment (DrZlo13) [reply applications/input/input_cli.c]: -_-

Comment (DrZlo13) [reply applications/input/input_cli.c]: I didn't look into input.c to find out. Done in b99e2bd.

**Review by skotopes — APPROVED** [review]

**Pull request merged**